release/tools/jaxrs.xml

```diff
@@ -19,7 +19,7 @@
 <project name="JAXRS" basedir="." default="build">
     
     <property name="deliverable.version"  value="3.0"/>
-    <property name="deliverable.tck.version"  value="3.0.0"/>
+    <property name="deliverable.tck.version"  value="3.0.1"/>
```

Comment:
Sorry, I should of mentioned that we already released [3.0.1 for EE 9 for a TCK challenge](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip).

Please increment again to 3.0.2

Reply:
@scottmarlow  - shall we merge of 90X branch changes to master? 
Compare - https://github.com/eclipse-ee4j/jakartaee-tck/compare/9.0.x

Reply:
@gurunrao yes and thank you!  I missed doing that before.  As per https://github.com/eclipse-ee4j/jakartaee-tck/projects/2, none of the challenged tests have been fixed yet so yes we should merge the 90x branch changes to master.

Reply:
Done.